libcxx/test/std/utilities/utility/mem.res/mem.res.monotonic.buffer/mem.res.monotonic.buffer.mem/release_reset_initial_status.pass.cpp

```diff
@@ -0,0 +1,52 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14
+// UNSUPPORTED: availability-pmr-missing
+
+// <memory_resource>
+
+// class monotonic_buffer_resource
+
+#include <memory_resource>
+#include <cassert>
+
+#include "count_new.h"
+#include "test_macros.h"
+
+int main(int, char**) {
+  {
+    {
+      // when init given a next buffer size, after release(), reset/not change next buffer size from initial state
+      constexpr auto expect_next_buffer_size{512ULL};
+      std::pmr::monotonic_buffer_resource mr{nullptr, expect_next_buffer_size, std::pmr::new_delete_resource()};
+
+      for (auto i{0}; i < 100; ++i) {
+        (void)mr.allocate(1);
+        mr.release();
+        ASSERT_WITH_LIBRARY_INTERNAL_ALLOCATIONS(globalMemCounter.checkLastNewSizeGe(expect_next_buffer_size));
+      }
+    }
+    {
+      // Given
+      // when init given a buffer, after release(), initial ptr will reset to it's initial state
```

Comment:
Sorry, I don't understand what this comment is trying to tell me. Could you formulate this in full sentences?

Reply:
I don't think you've addressed this.